.github/workflows/scripts/validate-form.py

```diff
@@ -155,7 +165,132 @@ def validate_form(issue_content_file: str) -> None:
+                        log.info(f"Received Legistar data at {days_prior} days")
+                        single_event = events[0].to_dict()
+                        event_as_json_str = json.dumps(single_event, indent=4)
+                        legistar_response = (
+                            f":x: Your municipality uses Legistar but the minimum "
+                            f"required data for CDP event ingestion wasn't found. "
+                            f"A "
+                            f"[cdp-scrapers]"
+                            f"(https://github.com/{COUNCIL_DATA_PROJECT}/cdp-scrapers) "
+                            f"maintainer should look into this issue however it is "
+                            f"likely that you (@{form_values[TARGET_MAINTAINER]}) will "
+                            f"need to write a custom scraper.\n"
+                            f"<summary>Retrieved Data</summary>\n"
+                            f"<details>\n\n"  # Extra new line for proper rendering
+                            f"```json\n"
+                            f"{event_as_json_str}\n"
+                            f"```"
+                            f"</details>"
+                        )
+                        break
+
+        # Catch no video path available
+        # User will need to write a custom Legistar scraper
+        except NotImplementedError:
+            legistar_response = (
+                f":warning: Your municipality uses Legistar but is missing the video "
+                f"URLs for event recordings. We recommended writing a custom Legistar "
+                f"Scraper that inherits from our own to resolve the issue. "
+                f"Please the "
+                f"[cdp-scrapers]"
+                f"(https://github.com/{COUNCIL_DATA_PROJECT}/cdp-scrapers) repository "
+                f"for more details. And please refer to the "
+                f"[SeattleScraper]"
+                f"(https://github.com/{COUNCIL_DATA_PROJECT}/cdp-scrapers"
+                f"/blob/main/cdp_scrapers/instances/seattle.py) for an example of a "
+                f"scraper that inherits from our base to resolve this issue."
+            )
+
+        except Exception:
+            legistar_response = (
+                f":x: Something went wrong during Legistar client data validation. "
+                f"A [cdp-scrapers]"
+                f"(https://github.com/{COUNCIL_DATA_PROJECT}/cdp-scrapers) maintainer "
+                f"should look into the logs for this bug. Sorry about this!"
+            )
+
+    # Handle bad / mis-parametrized legistar info
+    if legistar_response is None:
+        if (
+            form_values[LEGISTAR_CLIENT_ID] is not None
+            and form_values[LEGISTAR_CLIENT_TIMEZONE] is None
+        ):
+            legistar_response = (
+                ":x: You provided a Legistar Client Id but no Timezone. "
+                "**Timezone is required** for Legistar scraping."
```

Comment:
Might be nice to explicitly state here "Please edit your original submission/form/comment to include this information" or something like that.